src/initial-submission/components/FileDetailsForm.tsx

```diff
@@ -33,17 +33,47 @@ type UploadInProgress = {
 
 interface Props {
     initialValues?: Submission;
+    setIsSaving?: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
why is this optional? We can guarantee that this is always there so if we make it required we can then remove `toggleSave`